gcp/table_gcp_kubernetes_cluster.go

```diff
@@ -0,0 +1,423 @@
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "current_node_count",
+				Description: "The number of nodes currently in the cluster.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "current_node_version",
+				Description: "The current version of the node software components.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "status_message",
+				Description: "Additional information about the current status of this cluster, if available.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "subnetwork",
+				Description: "The name of the Google Compute Engine subnetwork to which the cluster is connected.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "enable_kubernetes_alpha",
+				Description: "Indicates whether kubernetes alpha features are enabled on this cluster.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "enable_tpu",
+				Description: "Enable the ability to use Cloud TPUs in this cluster.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "endpoint",
+				Description: "The IP address of this cluster's master endpoint.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "expire_time",
+				Description: "The time the cluster will be automatically deleted in RFC3339 text format.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("ExpireTime").Transform(transform.NullIfZeroValue),
+			},
+			{
+				Name:        "initial_cluster_version",
+				Description: "The initial Kubernetes version for this cluster.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "initial_node_count",
+				Description: "The number of nodes to create in this cluster.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "label_fingerprint",
+				Description: "The fingerprint of the set of labels for this cluster.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "logging_service",
+				Description: "The logging service the cluster should use to write logs.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "monitoring_service",
+				Description: "The monitoring service the cluster should use to write metrics.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "network",
+				Description: "The name of the Google Compute Engine network to which the cluster is connected.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "node_ipv4_cidr_size",
+				Description: "The size of the address space on each node for hosting containers.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "services_ipv4_cidr",
+				Description: "The IP address range of the Kubernetes services in this cluster, in CIDR notation.",
+				Type:        proto.ColumnType_STRING,
```

Comment:
Please validate the type
https://github.com/turbot/steampipe-plugin-gcp/pull/213/files#r695560137